### ISSUE TITLE
How was the model weights for Kitti obtained

### ISSUE THREAD

**Issue by Poulami-Sarkar** [hzxie/GRNet#49]
I am trying to reproduce the results of the paper on the Kitti Dataset. In your repository you have provided the linked for the pre-trained model for inference on the KITTI dataset. However, the KITTI dataset does not have the ground truth labels so I am confused on how this model was trained. 
Could you please let me know how this model was trained?

https://gateway.infinitescript.com/?fileName=GRNet-KITTI.pth

**Comment by hzxie** [comment]
It is trained with the car category of ShapeNet.
